config/initializers/exception_notification.rb

```diff
@@ -0,0 +1,14 @@
+require 'exception_notification/rails'
+require 'exception_notification/sidekiq'
+
+EXCEPTION_RECIPIENTS = JSON.parse(Figaro.env.exception_recipients).freeze
+
+ExceptionNotification.configure do |config|
+  config.add_notifier(
+    :email,
+    email_prefix: "[#{APP_NAME} EXCEPTION] ",
+    sender_address: %("Exception Notifier" <notifier@login.gov>),
```

Comment:
Should we i18n any of these, or make them configurable in application.yml?

Reply:
nah, doesn't seem worth it to to i18n? but it would probably be worth it to build the email address via the mail gem?

Reply:
Say more. I haven't used the mail gem.

Reply:
We can build this format of string programatically (which is useful if there are spaces or unicode)

```ruby
require 'mail'
=> true
Mail::Address.new('foo@bar.com').tap { |a| a.display_name = 'fooooo barrrr' }.to_s
=> "fooooo barrrr <foo@bar.com>"
```

Reply:
Maybe I'm misunderstanding, but is there a problem with the way the current sender address is implemented? I tested it with SES and it works fine. In what situations would the current implementation be problematic?

Reply:
In case we wanted to make the email configurable without interpolation or escaping? It's not a big deal.